src/public/sass/ITCSS/06_components/pages/category/_product.scss

```
@@ -0,0 +1,83 @@
+@use 'sass:color';
+@use '../../../01_settings/global' as *;
+@use '../../../01_settings/colors' as *;
+@use '../../../01_settings/fonts' as *;
+@use '../../../02_tools/breakpoints' as *;
+@use '../../../02_tools/mixins' as *;
+
+.product {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  padding: 1.875rem 0;
+  border: 0.0625rem solid color.adjust($color-aqua-blue, $lightness: 40%);
+  transition: border 100ms ease-in-out;
+  transition: box-shadow 500ms ease-in-out;
+
+  &__content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  &__image-container {
+    display: flex;
+    justify-content: center;
+  }
+
+  &__image {
+    max-width: 70%;
+    max-height: 17.5rem;
+    margin: 0 auto;
+    object-fit: cover;
+    transition: transform 300ms ease;
+
+    @include desktop {
+      max-width: 80%;
+    }
+  }
```

I think we can improve the code and aspect ratio of the images using this code @CarlosMay7 
```suggestion
  &__image {
    width: 70%;
    aspect-ratio: 10 / 12;
    object-fit: contain;
    transition: transform 300ms ease;
  }
```